pwiz_tools/Skyline/TestFunctional/RedundantComboBoxTest.cs

```diff
@@ -92,16 +93,24 @@ private static void VerifyRedundant(ViewLibraryDlg dlg, int i, bool visible, int
             RunUI(() => dlg.SelectedIndex = i);
             // The peptide at index one does not have redundant spectra
             int waitMs = 1000;
-            if (!TryWaitForConditionUI(waitMs, () => IsViewLibraryDlgState(dlg, i, visible, peakCount)))
+            if (!TryWaitForConditionUI(waitMs, () =>
```

Comment:
Should we put this code somewhere both tests can use it, like TestFunctionalEx? Maybe start a set of methods focused on driving the ViewLibraryDlg? Hate to see code that is this complex copied to multiple tests.

Reply:
I was planning on removing "TestRedundantComboBox" from "TargetedMsmsTutorialTest" because that's not actually part of the tutorial.

But, actually, I've thought of a better way to fix this: the "UpdateListPeptide" call in "ViewLibraryDlg_Shown" should pass in "listPeptide.SelectedIndex" to preserve the current selection in that big list box.